### ISSUE TITLE
Automatic bug reporting

### ISSUE THREAD

**Issue by game-stop** [game-stop/veejay#164]
Extend the crash handler to automatically create a new issue on github

- Each issue may be reported only once 
  (function, type of crash, line number optional)
- Throttle rate, github has a limitation on the number of issues you are allowed to create
- Ask the user for confirmation to send the bug-report

Related work: https://gist.github.com/JeffPaine/3145490
